rinja_derive/src/input.rs

```diff
@@ -422,6 +439,161 @@ impl TemplateArgs {
     }
 }
 
+#[cfg(feature = "code-in-doc")]
+/// Try to find the souce in the comment, in a "```rinja```" block
+///
+/// This is only done if no path or source was given in the `#[template]` attribute.
+fn source_from_docs(
+    name: &syn::Ident,
+    value: &syn::ExprLit,
+    dest: &mut Option<(Source, Option<Span>)>,
+    ast: &syn::DeriveInput,
+) -> Result<(), CompileError> {
+    let syn::Lit::Bool(syn::LitBool { value: true, .. }) = &value.lit else {
```

Comment:
In this case, please let's just check it's a boolean. And if `false`, we don't parse docs. It'd be a bit weird otherwise to expect only `true`. I suspect I will give it a try and sometimes just switch the boolean value to enable/disable the feature.

Reply:
Makes sense, done!